packages/coinstac-ui/app/render/components/consortia/consortia-list.jsx

```diff
@@ -175,25 +185,44 @@ class ConsortiaList extends Component {
 
     // Add owner/member list
     const ownersIds = consortium.owners.reduce((acc, user) => {
-      acc[user] = true;
+      let userID = Object.keys(user)[0];
```

Comment:
still wondering whats up here

Reply:
Users are being saved in consortia.members and consortia.owners arrays as an object with the userID as the key and the username as value. I thought this would be a simple way to store and reference members for UI display as opposed to having to query each user in Apollo (which can still be done).